src/main/java/com/gmail/nossr50/util/skills/SkillUtils.java

```diff
@@ -236,16 +241,50 @@ public static void handleDurabilityChange(ItemStack itemStack, int durabilityMod
      * @param durabilityModifier the amount to modify the durability by
      * @param maxDamageModifier the amount to adjust the max damage by
      */
-    public static void handleDurabilityChange(ItemStack itemStack, double durabilityModifier, double maxDamageModifier) {
+    public static void handleDurabilityChange(LivingEntity holder, ItemStack itemStack, double durabilityModifier, double maxDamageModifier) {
+        handleDurabilityChange(holder, itemStack, durabilityModifier, maxDamageModifier, false);
+    }
+
+    public static void applyDurabilityChange(Player player, ItemStack itemStack, int finalDurability) {
+        int damage = finalDurability - itemStack.getDurability();
+
+        // Call PlayerItemDamageEvent first to make sure it's not cancelled
+        if (damage > 0) {  // Damaged item
+            PlayerItemDamageEvent event = new PlayerItemDamageEvent(player, itemStack, finalDurability);
+            Bukkit.getPluginManager().callEvent(event);
+            if (!event.isCancelled() && event.getDamage() != 0) {
+                itemStack.setDurability((short) (itemStack.getDurability() + event.getDamage()));
+            }
+        } else if (damage < 0) {  // Repaired item
+            damage = -damage;
+            ExperienceOrb auxOrb = (ExperienceOrb) player.getWorld().spawnEntity(player.getLocation(), EntityType.EXPERIENCE_ORB);
```

Comment:
What's this experience orb stuff about? I don't get it.
Doesn't seem related to your PR.

Reply:
If the damage is negative, that means the item has been repaired. The PlayerItemMendEvent has an ExperienceOrb as an argument, so I needed to spawn one with the quantity of xp needed to repair the item for the specified amount if you had the mending enchantment. After the event has been called, the xp orb has to be despawned so no one can pick it up.
After your review I noticed I forgot to call the event, I just instantiate it and check if it has been canceled.

Reply:
I thought it would be better to call the mend event instead of the damage event when repairing an item, yet I think everything would also work with a damage event. I just did that for the sake of semantics.

Reply:
I see, thank you for the explanation, I will think on that some more.